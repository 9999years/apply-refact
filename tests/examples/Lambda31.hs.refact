[("tests/examples/Lambda31.hs:1:27-31: Suggestion: Redundant where\nFound:\n  where\nPerhaps you should remove it.\n",[])]
