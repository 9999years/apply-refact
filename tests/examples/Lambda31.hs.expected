f = \z -> foo $ z $ baz z where
